src/test/mmap_fd_reuse_checkpoint.c

```diff
@@ -0,0 +1,36 @@
+/* -*- Mode: C; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+
+#include "util.h"
+
+int gFd;
```

Comment:
just make this `static int fd`. We don't use this `g` stuff in rr :-).

Reply:
Hah, fun, that's one of the Gecko style rules I actually like, unlike the braces one ;)

Done anyhow :)